pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, makeWrapper
+, pciutils
```

Comment:
Unused.

Reply:
which ones exactly?

Reply:
That's a question for you to answer, not me.